libs/blocks/section-metadata/section-metadata.css

```diff
@@ -134,15 +134,30 @@ main>.section[class*='-up']>.content {
 .section.four-up,
 .section.five-up {
   display: grid;
-  grid-template-columns: repeat(auto-fit, minmax(276px, 1fr));
+  grid-template-columns: 1fr;
   gap: var(--spacing-m);
   align-items: start;
   padding-left: var(--grid-margins-width);
   padding-right: var(--grid-margins-width);
 }
 
```

Comment:
@Brandon32 
Sorry, I know this is coming late. I noticed that 900 media query doesn't allow very much viewport for the middle view where id expect a 2 up to be. ex. The original have 1-up mobile, 2-up tablet(600-1200, & *up on Desktop. Maybe thats ok? Just noticed a difference. 
Can we clean up the media queries so they are in order and at the bottom w/ no repeats?

```
/* mobile only */
@media (max-width: 600px) {
}

/* tablet up */
@media screen and (min-width: 600px) {
}

/* desktop up */
@media screen and (min-width: 1200px) {
}

/* desktop large < 1440 */
@media screen and (min-width: 1440px) {
}
```

Reply:
<img width="1849" alt="Screen Shot 2023-05-05 at 1 02 02 PM" src="https://user-images.githubusercontent.com/2086146/236547374-dbb0fee5-8fc5-410d-93bc-3066e17ff42c.png">

Reply:
2-up on tablet does not give a enough space for forms. 
![image](https://user-images.githubusercontent.com/681036/236572966-845bbebe-f389-4b11-8116-ed0fbf675337.png)

Our designs have 1-up on tablet.
![image](https://user-images.githubusercontent.com/681036/236572585-ed99bebe-522e-4920-be3b-7a1959b95d86.png)

Reply:
What about sticky-top? I'm not sure what it's used for.
~~~css
@media (min-width: 900px) {
  .section.sticky-top {
    top: 64px;
  }
}
~~~

Also there's this style:
~~~css
@media screen and (min-width: 600px) and (max-width: 1200px) {
  .section.five-up {
    grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
  }
}
~~~
Is `/* tablet up to desktop */` okay?

Reply:
@rgclayton I want to consolidate / standardize on a breakpoint since authors use block columns and section columns interchangeably.
Currently, below 700px two-up switches to one-up, and on columns switches to one-up below 622px.
I thought 900px was good a good point based on the designs having one-up at 768px and the current implementation at 700px being greater than 600px, I got but looks like I was incorrect.

Reply:
Could we use JS on the FaaS block to check if it is in the expected format (inside a text block, inside a `two up` section) and add a class to the parent section if `two up` is present? Like `faas-two-up` to protect prior blocks but get the expected layouts for FaaS two up?

Reply:
@ryanmparrish great point, as that pattern can cause unexpected behavior. Not sure why I didn't think of this, but just adding that same class to section metadata, though that puts more work on authoring.